doc/19_Development_Tools_and_Details/09_Cache/03_Full_Page_Cache.md

```diff
@@ -49,12 +49,43 @@ pimcore:
 Sometimes it is more useful to deactivate the full page cache directly in the code, for example when 
 it's not possible to define an exclude-regex, or for similar reasons.
 
-In this case you can use obtain the pull page cache service from the container and disable it, eg. in an action: 
+### Disable caching via the response headers
+Adding the `Cache-Control: no-store` header to your response will disable the full page cache as well as any other
+middleware and browser caching:
+```php
+<?php
+$response->headers->addCacheControlDirective('no-store');
+```
+
+### Disable the full page cache via an event listener
```

Comment:
This feature already existed in the full page cache service, but wasn't documented yet, so I added that as welll